11_child_other.do

```diff
@@ -6,10 +6,10 @@
 *c_ITN	Child slept under insecticide-treated-bednet (ITN) last night.
     gen c_ITN = .
 	
-	capture confirm variable ml0
```

Comment:
@yining-sun Thanks for the input, overall it looks good. However, replacing m10 to m1101 might not be a good move. Because there are cases where m10 is generating c_ITN for some surveys. This template you are working on is not only for just one survey-year but for all the AIS surveys. So it would be much better that do not delete the code that apply to one survey but don't apply to another, however to write the survey specific case in the condition command.  For example, instead of deleting, you can write ` if inlist(name, "cotedlvoire2005") {
replace c_ITN = (m1101 == 1)
replace c_ITN = . if m1101 == . } `

Reply:
Hi @yining-sun , I see you added the section of conditional command based on Survey year, thank you. Next time when you made changes based on the reviewer's comments, it would be fantastic if you could reply in the conversation chain accordingly to keep everyone posted of your actions.